src/v/kafka/server/rm_group_frontend.h

```diff
@@ -24,7 +24,8 @@ namespace kafka {
 
 ss::future<bool> try_create_consumer_group_topic(
   kafka::coordinator_ntp_mapper& mapper,
-  cluster::topics_frontend& topics_frontend);
+  cluster::topics_frontend& topics_frontend,
+  int16_t node_count);
```

Comment:
Maybe it is better to make it unsigned?

Reply:
The typing is a bit messy here:
- topic_configuration takes `int16_t` for replica count (not sure if it's actually ever allowed to be negative)
- internal_topic_replication_factor is an `int` (this is probably just wrong, it should be a uint16_t)
- the number of nodes from members manager is a size_t

I went with int16_t because it's the type we're going to want in the end for topic_configuration.

Reply:
Hmm, I dug into this a little more and there's a comment on topic_configuration saying that we use a signed int for Kafka protocol compatibility.  Really these should be uint16s, but if we did that they we'd have to handle the case of an overflow everywhere :-/